package/upgrade/migrations/upgrade_manifests/v1.4.2/pre-hook.sh

```diff
@@ -0,0 +1,504 @@
+#!/bin/bash -x
+
+apply_harvester_vm_migration_details_dashboard()
+{
```

Comment:
could you also check if the configmap is not existing then creat it?

Reply:
Sure, I already addressed the comment, thanks!